tests/e2e/testfiles/authorization-templates/csm-authorization_v1_storage.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: csm-authorization.storage.dell.com/v1alpha1
+kind: Storage
+metadata:
+  name: powerscale
+spec:
+  type: powerscale
+  endpoint: https://10.230.24.241:8080
```

Comment:
The spec values in the CRs should be updated during the test so users, and us, can pass in different values. This internal IP should be removed.

We can have uses put values in https://github.com/dell/csm-operator/blob/main/tests/e2e/array-info.sh and put them into the CRs.

Reply:
Right, missed that, thanks! Fixed.
Yeah, we can add populate them from that script, but I'd leave this change for another PR